array.go

```diff
@@ -107,6 +108,27 @@ func (a *Array) Err(err error) *Array {
 	return a
 }
 
+// Errs serializes and appends errors to the array.
```

Comment:
Centralized all the handling of Errs here, since everyone was calling `Arr()` and then doing the same things inline.